app/src/main/java/com/example/trainingdemoapp/utils/Constants.kt

```diff
@@ -0,0 +1,15 @@
+package com.example.trainingdemoapp.utils
+
+object Constants {
+    const val ZERO = 0
+    const val ONE = 1
+    const val TWO = 2
+    const val THREE = 3
+    const val FOUR = 4
+    const val FIVE = 5
+    const val EMPTY_STRING = ""
+    const val MSG = "msg"
+    const val BOTTOM_NAV_ITEM_HOME = "Home"
+    const val BOTTOM_NAV_ITEM_NOTIFICATIONS = "Notifications"
+    const val BOTTOM_NAV_ITEM_SETTINGS = "Settings"
+}
```

Comment:
make separate regions for different data type like int, string, long etc

Reply:
you can make internal classes to make different region.

Reply:
These are of no use anymore. Forgot to remove them.